manifests/cron.pp

```diff
@@ -22,10 +26,13 @@
     $scout_roles_switch = " -r ${roles}"
   }
 
+  $command = "/usr/bin/env scout \${SCOUT_KEY}${scout_environment_switch}${scout_roles_switch}"
```

Comment:
First thing I tried, as this would be neater!

Unfortunately, `join` converts `undef` to `"undef"` whereas string interpolation does not.